resources/lib/kodiutils.py

```diff
@@ -409,6 +410,24 @@ def get_global_setting(key):
     return result.get('result', {}).get('value')
 
 
+def get_property(key, window_id=10000):
+    ''' Get a Window property '''
+    from xbmcgui import Window
+    return Window(window_id).getProperty(key)
+
+
+def set_property(key, value, window_id=10000):
+    ''' Set a Window property '''
+    from xbmcgui import Window
+    return Window(window_id).setProperty(key, value)
```

Comment:
For Up Next I already improved upon this:
```suggestion
    return Window(window_id).setProperty(key, from_unicode(str(value)))
```